packages/discordeno/src/bin/generators/messageTypeGenerator.ts

```diff
@@ -0,0 +1,142 @@
+const messageTypeProps = {
```

Comment:
is this autogenerated or manually created? if autogenerated it shouldnt be in the repo, if manual, im not a fan of having this written twice here.